website/content/api-docs/allocations.mdx

```diff
@@ -1039,5 +1039,22 @@ $ nomad operator api /v1/client/allocation/23f520cc-629a-46ff-395f-0661e7aa939e/
 }
 ```
 
+#### Field Reference
```

Comment:
commenting here, though it mostly applies elsewhere.

this is the `ScheduleState` for _reads_, but for _writes_, there is a _different_ `ScheduleState`.  its values are listed over in the CLI doc: https://github.com/hashicorp/nomad/blob/v1.9.0/website/content/docs/commands/alloc/pause.mdx#pause-options (http server code [here](https://github.com/hashicorp/nomad/blob/v1.9.0/command/agent/alloc_endpoint.go#L463-L468))

1. do we want a reference section like this up under override on this page, with values that match `-state` in the CLI doc?
2. do we want the CLI doc to include more detail under `-status` to align with what's being added here?
3. maybe the CLI help output, too? https://github.com/hashicorp/nomad/blob/v1.9.0/command/alloc_pause.go#L36-L44

Reply:
I updated the POST/PUT API docs which I think is good for now.